### DISCUSSION TITLE
Which Godot version is the minimum requirement

### DISCUSSION THREAD

**Discussion by MikeSchulze** [MikeSchulze/gdUnit3#4] [FAQ] [answered]
GdUnit3 is based on Godot 3 this means the 3 in the name ;)

**Supported versions are**
3.3.x
3.4.x
3.5.x

**Not supported Versions**
lower than 3.3.1 and 4.x.x

**Comment by GeorgeS2019** [comment]
Once Godot4 dotnet6 merged, we may need to re-visit what are the supported versions e.g. in 6 months

**Reply by MikeSchulze** [reply, marked as answer]
For Godot4 a new project exists => https://github.com/MikeSchulze/gdUnit4
